### PR TITLE
added style and organized results

### PR DESCRIPTION
**Pull request by melanoke4**

### CODE REVIEW

**Comment by melanoke4** [comment]
adding style and organization

File: assets/food.js [hunk 1]
```diff
@@ -1,22 +1,7 @@
-// api variables
-var offset = 0
-var number = 10
-var apiKey = "ebd3c07a0d5542c69c8f71d07e4ac0f4"
-
```
Comment (KEHeidel) [inline assets/food.js]: Moved to different location

File: assets/food.js [hunk 2]
```diff
@@ -1,22 +1,7 @@
...
 // search bar (sliding)
-const trigger = document.querySelector(".trigger");
```
Comment (KEHeidel) [inline assets/food.js:7]: Renamed trigger to search button

File: assets/food.js [hunk 3]
```diff
@@ -1,22 +1,7 @@
...
-// trigger.addEventListener("click", () => {
-//   if (!input.classList.contains("input-open")) {
-//     // input.classList.add("input-open");
-//     trigger.innerHTML = "<i class='fas fa-search'></i>";
-//   } else {
-//     // input.classList.remove("input-open");
-//     trigger.innerHTML = "<i class='fas fa-times'></i>";
-//   }
-// });
-
```
Comment (KEHeidel) [inline assets/food.js]: removed uncommented code

File: assets/food.js [hunk 4]
```diff
@@ -34,13 +19,26 @@ input.addEventListener("keypress", function(event) {
   }
 });
 
-//  api key
```
Comment (KEHeidel) [inline assets/food.js:28]: changed comment

File: assets/food.js [hunk 5]
```diff
@@ -1,22 +1,7 @@
-// api variables
-var offset = 0
-var number = 10
-var apiKey = "ebd3c07a0d5542c69c8f71d07e4ac0f4"
-
 // search bar (sliding)
-const trigger = document.querySelector(".trigger");
+const trigger = document.querySelector(".searchButton");
```
Comment (KEHeidel) [inline assets/food.js:2]: Renamed trigger to search button

File: assets/food.js [hunk 6]
```diff
@@ -34,13 +19,26 @@ input.addEventListener("keypress", function(event) {
   }
 });
 
-//  api key
+// api variables
+var offset = 0;
+var number = 10;
+var apiKey = "ebd3c07a0d5542c69c8f71d07e4ac0f4";
+
+//  api key & function
```
Comment (KEHeidel) [inline assets/food.js:28]: API variables moved

File: assets/food.js [hunk 7]
```diff
@@ -49,55 +47,38 @@ function getrecipe(q) {
...
-  //     console.log(response);
-  //   });
-  // }
-  // getsource();
-  // function getrecipe(q) {
-  //   console.log(q);
-  //   offset = Math.floor(Math.random() * 100);
-  //   var recipes = ""
-  //   var queryURL = "https://api.spoonacular.com/recipes/search?query=" + q + "&offset=" + offset + "&number=" + number + "&apiKey=" + apiKey;
-  //   console.log("queryURL: " + queryURL);
```
Comment (KEHeidel) [inline assets/food.js]: Remove uncommented code

File: assets/food.js [hunk 8]
```diff
@@ -49,55 +47,38 @@ function getrecipe(q) {
...
-  //     for (var i = 0; i < res.results.length; i++) {
-  //       recipes += "<h1>" + res.results[i].title + "</h1><br><img src='" + res.baseUri + res.results[i].image + "'width='400'/><br> ready in " + res.results[i].readyInMinutes + "minutes";
-  //       // getsource(res.results[0].id);
-  //       console.log(res.results[i].title);
-  //     }
-  //     document.getElementById("output").innerHTML = recipes
-  //     console.log(res);
-  //   });
-  // }
-  $(".trigger").on("click", function(event) {
```
Comment (KEHeidel) [inline assets/food.js]: remove uncommented code

**Review by KEHeidel — APPROVED** [review]
reviewed code to push to master

**Comment by KEHeidel** [comment]
reviewed code to push to master

**Pull request merged**